script/.contributors_aliases.json

```diff
@@ -602,5 +602,9 @@
   "yileiyang@google.com": {
     "mails": ["yileiyang@google.com"],
     "name": "Yilei \"Dolee\" Yang"
+  },
+  "hofrob@protonmail.com": {
+    "mails": ["hofrob@protonmail.com"],
+    "name": "Robert Hofer"
```

Comment:
No need to do that unless you have another email or an aliased name you want to use. (Just for other people information, do not remove it now that it's done :)